cli.go

```diff
@@ -364,13 +366,17 @@ func printResult(out io.Writer, result *Result, mode DisplayMode, withStats bool
 	}
 
 	if withStats {
+		var timestampStr string
+		if verbose && !result.Timestamp.IsZero() {
+			timestampStr = fmt.Sprintf(" timestamp(%v)", result.Timestamp)
```

Comment:
How about using RFC3339 format with `result.Timestamp.Format(time.RFC3339Nano)`? We can copy and paste it for [TIMESTAMP function](https://cloud.google.com/spanner/docs/timestamp_functions) and also it can be used for upcoming staleness options https://github.com/cloudspannerecosystem/spanner-cli/issues/28.

Reply:
Nice catch!
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/30/commits/624e8676872456fa412f222ad96be1be21172a7f